extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -1646,3 +1651,38 @@ export class Bind {
     }
   }
 }
+
+/**
+ * Returns whether the element's property represents the value of a form field.
+ * @param {string} tagName
+ * @param {string} property
+ * @return {boolean}
+ */
+function isPropertyAFormValue(tagName, property) {
+  switch (property) {
+    case 'checked':
+    case 'value':
+      return tagName === 'INPUT';
+    case 'selected':
+      return tagName === 'OPTION';
+    case 'text':
+      return tagName === 'TEXTAREA';
+    default:
+      return false;
```

Comment:
Ah, I didn't realize there were so many cases. Might as well add `[disabled]` for form elements here too then. 

Then semantically this function becomes "any change, observable or not, that affects form data" instead of "only unobservable changes".